### PR TITLE
Fix string format error for snapshots.

### PR DESCRIPTION
**Pull request by mgjarrett**
## Description

Small string formatting bug fix. Attempted to format a string with two arguments but used the first argument in both places.
Bug introduced in #1239 

---

## Checklist

- [x] This PR has only one purpose or idea.
- [x] Tests have been added/updated to verify that the new/changed code works.

- [x] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [x] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [x] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any bug fixes or new features.
- [x] The documentation is still up-to-date in the `doc` folder.
- [x] The dependencies are still up-to-date in `setup.py`.

### CODE REVIEW

**Comment by john-science** [comment]
Looks good.  I see the original typo.

Why is this "Draft"?

File: doc/release/0.2.rst [hunk 1]
```diff
@@ -48,6 +48,7 @@ Bug fixes
 #. Fixed a bug in database comparison. (`PR#1258 <https://github.com/terrapower/armi/pull/1258>`_)
 #. Fixed an invalid assumption on the lattice physics and cross section manager interfaces when using tight coupling for snapshot runs. (`PR#1206 <https://github.com/terrapower/armi/pull/1206>`_)
 #. Fixed a bug where the precision used to determine the axial submesh was too small. (`PR#1225 <https://github.com/terrapower/armi/pull/1225>`_)
+#. Fixed a bug in the ISOTXS file name used for snapshots. (`PR#1277 <https://github.com/terrapower/armi/pull/1277>`_)
```
Comment (john-science) [inline doc/release/0.2.rst:51]: Damn, I was going to merge this PR.

But can you move this release note to the most recent version of ARMI (v0.2.8 above)?

Thanks! Sorry!

Comment (john-science) [reply doc/release/0.2.rst:51]: I would fix it myself, but you opened this PR from your private fork repo.

Comment (mgjarrett) [reply doc/release/0.2.rst]: Fixed, you can review now!

**Review by john-science — APPROVED** [review]

**Pull request merged**